tests/phpunit/tests/functions.php

```diff
@@ -1564,10 +1564,10 @@ public function test_wp_getimagesize_heic() {
 		}
 
 		$expected = array(
-			50,
-			50,
+			1181,
+			1181,
 			IMAGETYPE_HEIC,
-			'width="50" height="50"',
+			'width="1181" height="1181"',
```

Comment:
Seems this test may need to use [wp_fuzzy_number_match()](https://github.com/WordPress/wordpress-develop/blob/6.6.2/src/wp-includes/functions.php#L8858-L8860). It was made for these cases where image dimensions may be off by 1px because of processing.

Reply:
> Seems this test may need to use [wp_fuzzy_number_match()](https://github.com/WordPress/wordpress-develop/blob/6.6.2/src/wp-includes/functions.php#L8858-L8860). It was made for these cases where image dimensions may be off by 1px because of processing.

I resolved this by changing the order in which I processed the image. Earlier I resized then cropped, this time I cropped then resized. I must have had a pesky half pixel laying around last time.